TaskLayer/MyFileManager.cs

```diff
@@ -95,10 +96,15 @@ internal MsDataFile LoadFile(string origDataFile, int? topNpeaks, double? minRat
 
             // By now know that need to load this file!!!
             lock (fileLoadingLock) // Lock because reading is sequential
+            {
                 if (Path.GetExtension(origDataFile).Equals(".mzML", StringComparison.OrdinalIgnoreCase))
                 {
                     myMsDataFiles[origDataFile] = Mzml.LoadAllStaticData(origDataFile, filter);
                 }
+                else if (Path.GetExtension(origDataFile).Equals(".mgf"))
```

Comment:
`.Equals(".mgf", StringComparison.OrdinalIgnoreCase))`

Reply:
I've only ever seen .mgf lowercase, but I can add it.